lib/core/utils/get-ancestry.js

```diff
@@ -9,7 +9,7 @@ function generateAncestry(node) {
   if (
     nodeName !== 'head' &&
     nodeName !== 'body' &&
-    parentNode.children.length > 1
+    parentNode?.children.length > 1
```

Comment:
I didn't realize that the optional chaining operation short circuited like this; I got worried that this would need to be `parentNode?.children?.length`, but it turns out this is fine